setup.cfg

```diff
@@ -45,7 +45,7 @@ install_requires = dask
 		   numpy
 		   pandas
 		   portalocker
-                 pytest
+                   pytest
```

Comment:
Why does pytest look out of place when correct? I've had trouble since character one  🤷

Reply:
No idea, I've seen this in a few other places too.